tests/python/relay/test_tensorrt.py

```diff
@@ -0,0 +1,535 @@
+            run_and_verify(test_reduce(op, axis=(1, 2, 3), keepdims=keepdims))
+    run_and_verify(test_strided_slice((1, 3, 6, 7), (0, 0, 0, 0), (1, 1, 6, 7)))
+    run_and_verify(test_strided_slice((1, 3, 6, 7), (0, 1, 0, 0), (1, 2, 6, 6)))
+    run_and_verify(test_strided_slice((1, 10), (0, 0), (1, 10), (1, 2)))
+    for op in [relay.contrib.adaptive_max_pool2d, relay.contrib.adaptive_avg_pool2d]:
+        run_and_verify(test_adaptive_pool2d(op))
+        # run_and_verify(test_adaptive_pool2d(op, out_size=(6, 6)))
+    for x_shape, layout in [((1, 3, 16, 16), 'NCHW'), ((1, 16, 16, 3), 'NHWC')]:
+        for out_size in [(32, 32), (40, 40), (5, 21)]:
+            for method in ['nearest_neighbor', 'bilinear']:
+                for align_corners in [False]:
+                    # TODO(trevmorr): align_corners True gives incorrect results.
+                    run_and_verify(test_resize(x_shape, out_size, layout, method, align_corners))
+
+def test_tensorrt_integration():
+    if not tvm.module.enabled("cuda") or not tvm.gpu(0).exist:
+        print("skip because cuda is not enabled.")
+        return
+    if not tvm.get_global_func("relay.ext.tensorrt", True):
+        print("skip because tensorrt codegen is not available")
+        return
+    
+    def test_model(model, i_data, input_shape, dtype, use_trt=True, num_iteration=1000):
+        import mxnet
+        from mxnet.gluon.model_zoo.vision import get_model
+        def check_trt_used(graph):
+            import json
+            graph = json.loads(graph)
+            num_trt_subgraphs = sum([1 for n in graph['nodes'] if n.get('attrs', {}).get('func_name', '') == 'tensorrt_0'])
+            assert num_trt_subgraphs == 1
+
+        block = get_model(model, pretrained=True)
+        mod, params = relay.frontend.from_mxnet(block, shape={'data': input_shape}, dtype=dtype)
+
+        if use_trt:
+            mod = relay.transform.EnableTrt(mod, params)
+            assert mod['main'].attrs and mod['main'].attrs.Compiler == 'tensorrt'
+            with relay.build_config(opt_level=2, disabled_pass={"SimplifyInference"}):
+                graph, lib, params = relay.build(mod, "cuda")
+            check_trt_used(graph)
+        else:
+            with relay.build_config(opt_level=3):
+                graph, lib, params = relay.build(mod, "cuda", params=params)
+
+        mod = graph_runtime.create(graph, lib, ctx=tvm.gpu(0))
+        mod.set_input(**params)
+        # Warmup
+        for i in range(10):
+            mod.run(data=i_data)
+
+        # Time
+        times = []
+        for i in range(num_iteration):
+            start_time = time.time()
+            mod.run(data=i_data)
+            res = mod.get_output(0)
+            times.append(time.time() - start_time)
+        latency = 1000.0 * np.mean(times)
+        print(model, latency)
+        return latency, res
+
+    latency = {}
+    models = [
+        'alexnet',
+        'resnet18_v1',
+        # 'resnet34_v1',
```

Comment:
Either remove the commented models or put them in a `extra_mdoels` list and run them when a flag is enabled?